apps/publications-e2e/tsconfig.json

```diff
@@ -1,8 +1,5 @@
 {
   "extends": "../../tsconfig.base.json",
-  "compilerOptions": {
-    "types": ["cypress", "node"]
-  },
```

Comment:
I added this here I think because of some ts.lint problems, so I think It belongs here
![image](https://user-images.githubusercontent.com/32871167/109662467-61f38b00-7b6b-11eb-815b-c41340ffe943.png)
ng lint also founded some more problems